CMakeLists.txt

```diff
@@ -23,8 +23,9 @@ project(softwarecontainer)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules)
 
-# load GNUInstallDirs module
+# load some cmake modules
```

Comment:
I think this comment should explain why we use these modules, the fact that we import them is obvious.